ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -1,54 +1,82 @@
 <div>
-    <form method="post">
+    <form method="get">
         <div asp-validation-summary="ModelOnly"></div>
-        <div>
+        <div class="enter-search-details-tip">
+            To search enter at least one of the following fields
         </div>
-        <div>
-            <input type="submit" value="Search" />
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneQuarter">
+                @{
+                    var IdFilterError = !Model.IsValid("IdFilter");
+                    var IdFilterGroupState = IdFilterError ? Error : Standard;
+                }
+                <nhs-form-group nhs-form-group-type=@IdFilterGroupState v-bind:class="{ 'nhsuk-form-group--error': hasError}">
+                    <label nhs-label-type="Standard"> ID number </label>
+                    <span class="nhsuk-hint " id="example-hint">
+                        Search by NTBS, ETS, LTBR or NHS number
+                    </span>
+                    <span ref="errorField" nhs-span-type="ErrorMessage" id="id-error"
+                        asp-validation-for="IdFilter"></span>
+                    <input v-on:blur="validate" is-error-input=@IdFilterError nhs-input-type="Standard"
+                        asp-for="IdFilter" classes="nhsuk-input--width-10"
+                        v-bind:class="{ 'nhsuk-input--error': hasError}" />
+                </nhs-form-group>
+            </nhs-grid-column>
+        </nhs-grid-row>
+        <div class="submit-search-button-container">
+            <input type="submit" value="Search" class="nhsuk-button submit-search-button"/>
         </div>
     </form>
 </div>
-<div>
-    <div class="clear-search-button-container">
-        <a href="/search" class="clear-search-button">
-            Clear search & start again
-        </a>
-    </div>
+<hr />
+@if(Model.SearchParamsExist ?? false) 
+{
     <div>
-        <p> @Model.SearchResults.NumberOfResults Results found. </p>
-        <p> To filter results, add more search terms and search again. </p>
-    </div>
-    <ol class="search-results-list">
-        @foreach(var result in @Model.SearchResults) {
-            <li>
-                <partial name="../Shared/_NotificationBanner.cshtml" model="result"/>
-            </li>
+        <div class="clear-search-button-container">
+            <a href="/search" class="clear-search-button">
+                Clear search & start again
+            </a>
+        </div>
+        <div>
+            @{ var resultCount = Model.SearchResults?.NumberOfResults ?? 0; }
+            <p> @resultCount Results found. </p>
+            <p> To filter results, add more search terms and search again. </p>
+        </div>
+        @if(Model.SearchResults != null) 
+        {
+            <ol class="search-results-list">
+                @foreach(var result in @Model.SearchResults) 
+                {
+                    <li>
+                        <partial name="../Shared/_NotificationBanner.cshtml" model="result"/>

```

Comment:
This might be confusing at the beginning for those reading the code, I mean showing Banner as a search result. Do you think we should add a small comment explaining it?